bioio_base/reader.py

```diff
@@ -194,6 +195,26 @@ def current_scene_index(self) -> int:
         """
         return self._current_scene_index
 
+    @property
+    def resolution_levels(self) -> Tuple[int, ...]:
+        """
+        Returns
+        -------
+        resolution_levels: Tuple[int, ...]
+            A tuple of valid resolution levels in the file.
+        """
+        return (self._current_resolution_level,)
+
+    @property
+    def current_resolution_level(self) -> int:
+        """
+        Returns
+        -------
+        resolution_level: int
+            The current resolution level.
+        """
+        return self._current_resolution_level
+
```

Comment:
It seems backward to me to initialize a current resolution level and then build resolution levels from that initialization.

Reply:
I might be misunderstanding what you are getting at, but the reason the `_current_resolution_level` property has to exist in conjunction to a property of similar name (`current_resolution_level`) is because the former is protected from getting set by users outside of the way we want (as in we don't want `_current_resolution_level = 3` and do want `set_resolution_level(3)`) and the latter provides an access point to users. We could do with just the private `_current_resolution_level` but generally you don't want to build an API expecting users to have to access a private property.